src/app/pages/news/11.3/02-inscription-view.md

```diff
@@ -1,6 +1,22 @@
 ## Inscription View {#inscriptionView}
 
-Massive design update, browser, finalization ...
+The new Inscription View is finally finished! 
+We've not only successfully migrated all the old SWT-based 
+masks and browsers into the new world but have also given the entire 
+interface an amazing redesign.
```

Comment:
not sure whether we should talk about 'old SWT-based' ... most users do not know, that the legacy inscriptions are built with SWT. 
furthermore I'd not call anything old but keep talking about the fresh good new stuff.
e..g 
> The new Inscription View is finally feature complete! You are able to configure all elements on the totally redesigned interface.